site/docs/get-started/quickstart.md

```diff
@@ -44,7 +44,7 @@ Many clusters already include cert-manager. Check if `cert-manager` namespace ex
 ```bash
 helm install cert-manager jetstack/cert-manager \
     --namespace cert-manager \
-    --version v1.2.0 \
+    --version v1.6.0 \
```

Comment:
should this be 1.6.2?

Reply:
Right. this PR does not include documentation changes yet.